packit_service/worker/whitelist.py

```diff
@@ -215,14 +214,16 @@ def check_and_report(self, event: Optional[Any], project: GitProject) -> bool:
             if not account_name:
                 raise KeyError(f"Failed to get account_name from {type(event)}")
 
-            if not self.is_approved(account_name):
-                logger.error(f"User {account_name} is not approved on whitelist!")
+            namespace = event.base_repo_namespace
+            if not (self.is_approved(account_name) or self.is_approved(namespace)):
+                msg = f"Account {account_name} or project {namespace} are not whitelisted!"
```

Comment:
`event.base_repo_namespace` contains either [event["pull_request"]["base"]["repo"]["owner"]["login"]](https://github.com/packit-service/packit-service/blob/master/packit_service/worker/parser.py#L145) or [event["repository"]["owner"]["login"]](https://github.com/packit-service/packit-service/blob/master/packit_service/worker/parser.py#L233) (i.e. owner)

```suggestion
                msg = f"Account {account_name} and owner {namespace} are not whitelisted!"
```

Reply:
so account_name is namespace then?

Reply:
`account_name`, i.e. (`event.github_login`) comes from `event["pull_request"]["user"]["login"]`, i.e. it's the user which caused the event (created PR, added a comment, etc.)